build.gradle

```diff
@@ -24,6 +24,26 @@ if (ext.lcovIgnore == null) {
     ext.lcovIgnore = 'source'
 }
 
+def getDescriptionForPom(isFips) {
+    if (!isFips) {
+        return [
+            'The Amazon Corretto Crypto Provider (ACCP) is a collection of',
+            'high-performance cryptographic implementations exposed via the',
+            'standard JCA/JCE interfaces.'
+        ].join(" ")
+    }
+
+    return [
+        'The Amazon Corretto Crypto Provider FIPS (ACCP-FIPS) is a collection',
+        'of high-performance cryptographic implementations exposed via the',
+        'standard JCA/JCE interfaces. ACCP-FIPS uses AWS-LC built in FIPS mode',
+        'and is provided for experimental purposes only.'
+    ].join(" ")
+
+}
+
+println getDescriptionForPom(ext.isFips)
```

Comment:
Was this leftover from development? Suggest dropping it as we don't really log this kind of stuff anywhere else.

Reply:
You got me. I'll remove those.

Reply:
done